### ISSUE TITLE
minko master html5: issue with mouse movement

### ISSUE THREAD

**Issue by fjallraven** [aerys/minko#200]
Having issues getting mouse dx/dy movement
This can be seen in the "light" example - built as html5

It does no longer work to rotate the camera around

Thank you

**Comment by Noxalus** [comment]
Hello fjallraven,
I had the same issue 2 weeks ago. You can fix it replacing the line _498_ of [**Canvas.cpp**](hhttps://github.com/aerys/minko/blob/master/plugin/sdl/src/minko/Canvas.cpp#L498) (from SDL plugin):
```
_mouse->move()->execute(_mouse, x - oldX, y - oldY);
```
by this.
```
_mouse->move()->execute(_mouse, event.motion.xrel, event.motion.yrel);
```
(You can also remove *oldX* and *oldY* variables, they are not needed anymore).

This should be enough to resolve your problem. :)

**Comment by fjallraven** [comment]
Many thanks! That did the trick

**Comment by warrenseine** [comment]
Fixed on master (1f47af99af8a29bc67364566ce8ac4cb79743ea2).
